velox/exec/Task.cpp

```diff
@@ -1753,7 +1756,8 @@ void Task::addCustomJoinBridgesLocked(
   auto& splitGroupState = splitGroupStates_[splitGroupId];
   for (const auto& planNode : planNodes) {
     if (auto joinBridge = Operator::joinBridgeFromPlanNode(planNode)) {
-      splitGroupState.bridges.emplace(planNode->id(), std::move(joinBridge));
+      splitGroupState.custom_bridges.emplace(
```

Comment:
Add a check to make sure there is no previous value on the same node ID.  Same change to  `addHashJoinBridgesLocked`.

Reply:
Added it to `addHashJoinBridgesLocked`, `addCustomJoinBridgesLocked`, `addNestedLoopJoinBridgesLocked`